.github/workflows/sdk-release.yml

```diff
@@ -93,7 +93,7 @@ jobs:
       fail-fast: false
       max-parallel: 1
       matrix:
-        repo: [block-explorer, 0nft, 0box, 0dns, 0proxy, zwalletcli, zboxcli, zboxmobile, blobber ]
+        repo: [0nft, 0box, 0dns, zwalletcli, zboxcli, blobber, zs3server, s3-migration ]
```

Comment:
gosdk is also used in [authorizers](https://github.com/0chain/token_bridge_authserver/blob/staging/code/go/0chain.net/authorizer/go.mod#L6)

Reply:
web-apps should be upgraded. @ranajahanzaib  could you copy it from block-explorer?

Reply:
added `token_bridge_authserver` repo to update gosdk